[ISSUE] run make release.verify-components-version on MacOS failed

**Describe the issue:**

<!-- A clear and concise description of what the issue is. -->
when I run the make target on MacOS, I get the error:
```sh
➜  camunda-platform-helm git:(main) ✗ make release.verify-components-version
"camunda" already exists with the same configuration, skipping
Hang tight while we grab the latest from your chart repositories...
...Successfully got an update from the "elastic" chart repository
...Successfully got an update from the "kusionstack" chart repository
...Successfully got an update from the "ingress-nginx" chart repository
...Successfully got an update from the "camunda" chart repository
...Unable to get an update from the "loft-sh" chart repository (https://charts.loft.sh):
        read tcp 192.168.3.9:52264->34.121.62.77:443: read: connection reset by peer
Update Complete. ⎈Happy Helming!⎈
grep: invalid option -- P
usage: grep [-abcdDEFGHhIiJLlMmnOopqRSsUVvwXxZz] [-A num] [-B num] [-C[num]]
        [-e pattern] [-f file] [--binary-files=value] [--color=when]
        [--context[=num]] [--directories=action] [--label] [--line-buffered]
        [--null] [pattern] [file ...]
make[1]: *** [release.verify-components-version] Error 2
```

**Expected behavior:**
execute the make target successfully.

**How to reproduce:**
➜  camunda-platform-helm git:(main) ✗ make release.verify-components-version

<!--
Steps to reproduce the issue.

If possible add a minimal reproducer code sample in a new repo/branch.
-->

**Logs:**

<!-- If possible add the full logs related to the issue. -->

**Environment:**

**Please note: Without the following info, it's hard to resolve the issue and probably it will be closed.**

- Platform: <!-- [e.g. GCP, AWS, etc] -->
- Helm CLI version: <!-- [e.g. 3.10.0] -->
- Chart version: master
- Values file: <!-- [e.g. include or link to your values file] -->

--- a/scripts/verify-components-version.sh
+++ b/scripts/verify-components-version.sh
@@ -10,7 +10,10 @@ helm repo update
 
 chart_main_dir=$(ls -d1 charts/camunda-platform-8* | tail -n1)
 chart_main_version="$(yq '.version' ${chart_main_dir}/Chart.yaml)"
-components_versions="$(helm template camunda/camunda-platform | grep -Po '(?<=helm.sh/chart: ).+' | sort | uniq)"
+components_versions=$(helm template camunda/camunda-platform | \
+  awk -F'helm.sh/chart: ' '/helm.sh\/chart:/ {print $2}' | \
+  sort | uniq)
+
 components_count=2
 
 print_components_versions() {

--- a/scripts/verify-components-version.sh
+++ b/scripts/verify-components-version.sh
@@ -10,9 +10,7 @@ helm repo update
 
 chart_main_dir=$(ls -d1 charts/camunda-platform-8* | tail -n1)
 chart_main_version="$(yq '.version' ${chart_main_dir}/Chart.yaml)"
-components_versions=$(helm template camunda/camunda-platform | \
-  awk -F'helm.sh/chart: ' '/helm.sh\/chart:/ {print $2}' | \
-  sort | uniq)
+components_versions=$(helm template camunda/camunda-platform | awk -F'helm.sh/chart: ' '/helm.sh\/chart:/ {print $2}' | sort | uniq)
 
 components_count=2
 

